automation/test-output/Default suite/Default test.html

```diff
@@ -0,0 +1,84 @@
+<html>
```

Comment:
Add the test-output folder to .gitIgnore (don't include this folder in the repo.)